src/librustdoc/html/render/mod.rs

```diff
@@ -2344,9 +2355,49 @@ fn sidebar_trait(cx: &Context<'_>, buf: &mut Buffer, it: &clean::Item, t: &clean
     buf.push_str("</section>")
 }
 
+pub(crate) fn get_reference_impls<'a>(
```

Comment:
nit: get_filtered_impls_for_reference, and add a doc comment saying what this function does